rpcs3/rpcs3qt/main_window.cpp

```diff
@@ -466,6 +474,11 @@ void main_window::InstallPkg(const QString& dropPath, bool is_bulk)
 		LOG_SUCCESS(GENERAL, "Successfully installed %s.", fileName);
 		guiSettings->ShowInfoBox(tr("Success!"), tr("Successfully installed software from package!"), gui::ib_pkg_success, this);
 	}
+	else if (!cancelled)
+	{
+		LOG_ERROR(GENERAL, "Failed to install %s.", fileName);
+		QMessageBox::critical(this, tr("Failure!"), tr("Failed to install software from package!"));
```

Comment:
This is not really transparent to the user since it might be a bulk installation.
So which software pkg failed to install ?

Reply:
Ah, wasn't aware of a bulk installation thing. Will fix.